lib/robots/dor_repo/speech_to_text/fetch_files.rb

```diff
@@ -12,14 +12,20 @@ def initialize
         # available from LyberCore::Robot: druid, bare_druid, workflow_service, object_client, cocina_object, logger
         def perform_work
           sttable_filenames.each do |filename|
-            raise "Unable to fetch #{filename} for #{druid}" unless file_fetcher.write_file_with_retries(filename:, location: aws_provider.bucket.object(File.join(bare_druid, filename)), max_tries: 3)
+            raise "Unable to fetch #{filename} for #{druid}" unless file_fetcher.write_file_with_retries(filename:, location: aws_provider.bucket.object(File.join(job_id, filename)), max_tries: 3)
           end
         end
 
         private
 
         def sttable_filenames
-          Dor::TextExtraction::SpeechToText.new(cocina_object:, workflow_context: workflow.context).filenames_to_stt
+          Dor::TextExtraction::SpeechToText.new(cocina_object:).filenames_to_stt
```

Comment:
the `filenames_to_stt` method doesn't need workflow_context so don't bother passing it in